packages/app/src/cli/models/app/identifiers.ts

```diff
@@ -91,9 +91,7 @@ export function getAppIdentifiers(
       extensionsIdentifiers[extension.localIdentifier] = envVariables[extension.idEnvironmentVariableName]!
     }
   }
-  app.extensions.ui.forEach(processExtension)
-  app.extensions.function.forEach(processExtension)
-  app.extensions.theme.forEach(processExtension)
+  app.allExtensions.forEach(processExtension)
```

Comment:
We're losing the ordering of tackling ui, then function, then theme. Any downside to that?

Reply:
nop, there is no reason to have a specific order here